internal/pkg/service/buffer/store/receiver.go

```diff
@@ -17,7 +18,7 @@ import (
 // Logic errors:
 // - CountLimitReachedError
 // - ResourceAlreadyExistsError.
-func (s *Store) CreateReceiver(ctx context.Context, receiver model.Receiver) (err error) {
+func (s *Store) CreateReceiver(ctx context.Context, receiver model.Receiver, files []*storageapi.File) (err error) {
```

Comment:
Vytvaranie file/slice modelu je v pohode, ale toto spajanie `file->export` cez index pola mi trocha smrdi.

Hlavny problem je teda asi, ze `Export` nema field `Files`, ktorym by sme to tu dostali. A `File` zase nema `Slices`.

Su teda 2 moznosti:
- `store` by mohol volat aj API na vytvorenie suboru, ale tym by uz mal viacero zodpovednosti, co asi nechceme
- namiesto slice pouzit `map[key.ExportKey]*storageapi.File` co mi pride v pohode, co myslis?

Teda nestane sa ti, ze nejaky index sa neprida a pomiesaju sa ti subory pre rozne exports.
Ak kluc neexistuje, tak mozeme vyhodit `panic`, kedze to bude nejaka chyba vyvojara.